### ISSUE TITLE
text within the gray bar at the top of the page is not contained in a landmark

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2515]
## Source

Pacific ADA

## Describe the bug

The text within the gray bar at the top of the page -  The text from "2024 Americans with Disabilities Act...January 8-11 - is not contained in a landmark.   

The text within it is further not contained in a semantic element.  Semantic markup provides context for screen readers. 

All perceivable text content should be included within an aria landmark.   This is a best practice only and not currently required for WCAG conformance.  WCAG 2.1 

[Technique ARIA11: Using ARIA landmarks to identify regions of a page](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11)

https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA20.html

## Recommendations

Example of semantic updates:

1.) Put the text in a <p> tag.
2.) Add role="region" and aria-label="Important Notifications".

## Screenshots

![semantic updates](https://github.com/proudcity/wp-proudcity/assets/1569463/3201b7fe-90df-4b2f-b349-338b1ef4a02c)
